src/Revit/RevitHyparTools/Utils.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Autodesk.Revit.DB;
+using Elements.Geometry;
+
+namespace Hypar.Revit
+{
+    public static class Utils
+    {
+        public static Elements.Geometry.Profile ScaleProfileFtToMeters(Elements.Geometry.Profile profile) {
+            var transform = new Elements.Geometry.Transform();
+            transform.Scale(Elements.Units.FeetToMeters(1));
+            var transformedProfile = transform.OfProfile(profile);
+            
+            return new Elements.Geometry.Profile(transformedProfile.Perimeter, transformedProfile.Voids, Guid.NewGuid(), transformedProfile.Name);
+        }
+        public static Elements.Geometry.Profile ReverseProfile(Elements.Geometry.Profile profile)
+        {
+            var perimeter = profile.Perimeter.Reversed();
+            var voids = profile.Voids.Select(v => v.Reversed());
+
+            return new Elements.Geometry.Profile(perimeter, voids.ToList(), Guid.NewGuid(), "");
+        }
+
+        // Revit stores lengths in ft, but Hypar's standard is meters, so we need to scale the faces when we convert them to profiles
+        public static Elements.Geometry.Profile[] GetScaledProfilesOfFace(PlanarFace f)
```

Comment:
Extension method: `PlanarFace.GetProfiles(...)` The fact that the profiles are scaled is implied in the conversion to Hypar geometry.

Reply:
great, done